hallebarde-frontend/.env

```diff
@@ -1,6 +1,6 @@
-REACT_APP_USER_POOL_ID="eu-west-1_v0laOwH0X"
-REACT_APP_USER_POOL_WEB_CLIENT_ID="472ko5t276u2mdrl29rnstl6qj"
-REACT_APP_OAUTH_DOMAIN="hallebarde-dev.auth.eu-west-1.amazoncognito.com"
-REACT_APP_OAUTH_CALLBACK_URL="https://dev.bda.ninja/callback"
-REACT_APP_OAUTH_SIGNOUT_URL="https://dev.bda.ninja/signout"
-REACT_APP_BACKEND_URL="dev.api.bda.ninja"
\ No newline at end of file
+REACT_APP_USER_POOL_ID="eu-west-1_ChoVjs2Xy"
+REACT_APP_USER_POOL_WEB_CLIENT_ID="78m0k0dgc08hnequo7furn0qb7"
+REACT_APP_OAUTH_DOMAIN="dev-filetransfer.auth.eu-west-1.amazoncognito.com"
+REACT_APP_OAUTH_CALLBACK_URL="http://localhost:3000/callback"
+REACT_APP_OAUTH_SIGNOUT_URL="http://localhost:3000/signout"
```

Comment:
Does this variable have an effect on production side ? (I ask because of the `localhost` part), and this .env file seems to be environment agnostic as it is at the root folder

Reply:
actually it is has an effect, and it is not agnostic. This `.env` file is the mecanism I found for building the app before upload to the S3 hosting with the right OAuth information in it (Cognito endpoints, redirect URIs, etc.).

The thing being that because S3 offers only _static_ hosting, we cannot dynamically process these information, and they have to be embedded into the app.

You can see a `configure-environment.sh` file at the root. If you look at the `Makefile`, you will see how it is used, I suppose we can make this more explicit?
Basically, it gets the terraform output in order to fill the `.env` file with the right information, depending on the env you indicate: local, dev, or prod. (Local uses the dev cognito instances, but with redirect URIs pointing to localhost).